pywbem_mock/_baserepository.py

```diff
@@ -0,0 +1,409 @@
+#
+# (C) Copyright 2020 InovaDevelopment.comn
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#
+# Author: Karl  Schopmeyer <inovadevelopment.com>
+#
+
+
+"""
+Base abscract classes for an object store for collections of
+:class:`~pywbem.CIMClass`, :class:`~pywbem.CIMInstance`, and
+:class:`~pywbem.CIMQualifierDeclaration`s and a generic API for a CIM
+repository to access and manage the collections of each of these CIM object
+types.
```

Comment:
Even though it is not part of the generated docs:
* Typo: "abscract"
* don't say "`CIMQualifierDeclaration`s" but instead "`CIMQualifierDeclaration` objects"

Reply:
Fixed spelling and remove the floating  in favor of word objects.

Reply:
You fixed only item 2 from the list, but not item 1 (the typo).